pkg/webhooks/namespace/namespace.go

```diff
@@ -237,17 +226,9 @@ func (s *NamespaceWebhook) authorized(request admissionctl.Request) admissionctl
 		ret.UID = request.AdmissionRequest.UID
 		return ret
 	}
-
-	// If the user making the request has a specific exception, allow them to change labels on non-platform and non-protected namespaces
-	if allowLabelChanges(request) {
-		ret = admissionctl.Allowed("User allowed to modify namespace labels")
-		ret.UID = request.AdmissionRequest.UID
-		return ret
-	}
-
-	// Unprivileged users cannot modify certain labels on unprivileged namespaces
+	// Check labels.
 	unauthorized, err := s.unauthorizedLabelChanges(request)
-	if unauthorized {
+	if !amIAdmin(request) && unauthorized {
```

Comment:
I am trying to understand why we add `!amIAdmin(request)` in this condition. It seems line 186 should already checked `amIAdmin(request)`? or is there anything specific here?